guides/08_Migration_to_2_0.md

```diff
@@ -174,23 +186,26 @@ class MyDeviceController implements DeviceController {
 }
 ```
 
-At present `VideoInputDevice` is identical to `Device`, and so the only change you need to make is to change the parameter type of `chooseVideoInputDevice` from `Device` to `VideoInputDevice`.
+A similar adjustment is needed for `VideoInputDevice`. v2.0 does not include any
+`VideoTransformDevice`s.
 
 If you use the type `Device` for a field or variable and pass that value to
 `choose{Audio,Video}InputDevice`, your code should continue to work without changes.
 
 ---
 
-## Introducing `GetUserMediaError` type
-With version 2.0, we are introducing new error types which will be thrown in case of any failures in `chooseAudioInputDevice` and `chooseVideoInputDevice` APIs in the `DeviceController`.
+### Introducing the `GetUserMediaError` type
+
+v2.0 introduces new error types which will be thrown in case of any failures in `chooseAudioInputDevice` and `chooseVideoInputDevice` APIs in the `DeviceController`.
 
 Here is the list of new error classes introduced in version 2.0:
-- `GetUserMediaError`
-- `NotFoundError`
-- `NotReadableError`
-- `OverconstrainedError`
-- `PermissionDeniedError`
-- `TypeError`
+
+* `GetUserMediaError`
```

Comment:
nit: formatting

Reply:
Could you clarify, @vidya-mohan? The linter specifically asks for `*` rather than `-`, and I indented the subclasses underneath the parent `GetUserMediaError` class, because it shows the inheritance relationship.